### PR TITLE
dev: added simple smoke test for preview environment (CMC-NA)

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

N/A

### Change description ###

Added simple login smoke test for solicitor user

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by swalker125** [comment]
From pipeline 

[2020-06-29T13:19:06.210Z]   Smoke tests @smoke-tests

[2020-06-29T13:19:06.210Z] Smoke tests @smoke-tests --

[2020-06-29T13:19:09.503Z]     ✓ Sign in as solicitor user (4431ms)

[2020-06-29T13:19:09.503Z] 
  ✔ Sign in as solicitor user in 4431ms

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "browser": true,
+    "commonjs": true,
+    "es2020": true,
+    "codeceptjs/codeceptjs": true
+  },
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 11
+  },
+  "rules": {
```
Comment (pd-dudkiewicz) [inline .eslintrc.js:15]: how about adding
```
"semi": [
  "error",
  "always"
]
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (pd-dudkiewicz) [inline package.json:1]: should we add `yarn lint` as pre-push hook?

File: e2e/tests/smoke_test.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+/* global process */
+
+const user = {
+  password: 'Password12',
+  email: 'solicitor@example.com'
+}
```
Comment (pd-dudkiewicz) [inline e2e/tests/smoke_test.js]: I'd put it in separate config file already

**Review by pd-dudkiewicz — DISMISSED** [review]
few minor comments but happy to approve :+1:

File: .eslintrc.js (hunk 1 above)
Comment (swalker125) [reply .eslintrc.js:15]: sure, there are more rules I am sure we can add too as we go

File: package.json (hunk 2 above)
Comment (swalker125) [reply package.json:1]: happy to add, saves pushing broken stuff hopefully 😅

File: e2e/tests/smoke_test.js (hunk 3 above)
Comment (swalker125) [reply e2e/tests/smoke_test.js]: makes sense

**Review by pd-dudkiewicz — DISMISSED** [review]

**Pull request merged**